libzhlgen/test/functions/ELF_amd64/1.6.13/SpaceStatus.zhl

```diff
@@ -0,0 +1,6 @@
+"415741564155415449bc0000":
+cleanup __amd64 void SpaceStatus::RenderWarningText(SpaceStatus *this, int effect, int textOffset);
+"415455534889fb4883ec10488b7f58e8":
+cleanup __amd64 void SpaceStatus::OnRender(SpaceStatus *this);
+"41544189d45589f5534889fb4883ec20":
```

Comment:
All new 64-bit signatures except the one at the top of the file should be relative if possible

Reply:
Also I have not validated these signatures yet